koku/masu/api/upgrade_trino/util/__init__.py

```diff
@@ -0,0 +1,6 @@
+CONTEXT_KEY_MAPPING = {
+    "metadata": "conversion_metadata",
+    "version": "version",
+    "successful": "successful",
+    "failed_files": "dtype_failed_files",
+}
```

Comment:
This seems like a good candidate for [`StrEnum`](https://docs.python.org/3/howto/enum.html) since it is storing unique constant values..

```suggestion
from enum import auto
from enum import Enum
class Context(StrEnum):
    metadata = "conversion_metadata"
    version = auto()
    successful = auto(),
    failed_files = "dtype_failed_files"
```

In use, it would look like `bill_metadata.get(Context.version)`. 

`StrEnum` was added in Python 3.11, but it's a [fairly simple class](https://github.com/python/cpython/blob/c31be58da8577ef140e83d4e46502c7bb1eb9abf/Lib/enum.py#L1321-L1345) we could vendor.

Reply:
Great suggestion @samdoran. I wasn't able to get the `auto()` to work because it was generating nums and StrEnum expected stings. However, I did go a head and switch my file states over to StrEnums too.

Reply:
I was looking through the CPython source and there were a number of changes to the metaclass to make `auto()` work. Explicitly setting the value works just fine.